tests/test_e2e/test_cloud_spec.py

```diff
@@ -0,0 +1,58 @@
+import ops
+import pytest
+
+import scenario
+
+
+class MyCharm(ops.CharmBase):
```

Comment:
I _think_ you can get rid of MyCharm and simply do:

`ctx = Context(ops.CharmBase, meta = ...)`

Reply:
I prefer this way, two reasons:

1. It's consistent with other e2e test files where a charm is defined as a class.
2. It's how end users would use scenario cloud spec: they have a charm as a class.

Reply:
fine by me, also easier to extend it if we want to add some stuff in the charm in the future.